public/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <!-- <link rel="stylesheet" href="/style.css"> -->
+  <title>FEC Demo - BLITVA-Amazon</title>
+</head>
+
+<body>
+  <h1>Amazon</h1>
+  <div id="photos"></div>
+  <div id="CustomerQuestions"></div>
+  <div id="CustomerReviews"></div>
+  <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+  <script src="http://localhost:4002/photos_bundle.js"></script>
+  <script src="http://localhost:4001/customerQuestions_bundle.js"></script>
+  <script src="http://localhost:4006/reviews_bundle.js"></script>
```

Comment:
I see that you are displaying three services onto the page. I was just wondering if you had trouble displaying my description service or if there was any specific reason for only having these three services?

Reply:
Hi Amos , I was running into error and I might have the old version of the code. It was already late and I had to submit the video, so went ahead with just few. I will be trying it out today for remaining ones after getting latest and will let you know.